pyam/utils.py

```diff
@@ -428,6 +428,11 @@ def find_depth(data, s="", level=None):
         whether depth satisfies the condition (equality if level is int,
         >= if ``.+``,  <= if ``.-``)
     """
+    if type(level) == list:
```

Comment:
Maybe better? From `utils.py`...
```suggestion
    if islistable(level):
```